### DISCUSSION TITLE
How to change Icecast URLs after domain name change?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6128] [Q&A] [answered]
I’m using Rolling Release #[5cbf9c9](https://github.com/AzuraCast/AzuraCast/commit/5cbf9c91cae1a234c428cf43d769fe2349e05bd9) (2023-02-28 8:15) and had to change my domain (now a subdomain of one of my real Internet domains).

I changed the Site Base URL in System Settings, and also the Web Site URL of the station, then did a complete restart using
```
docker-compose down
docker-compose up -d
```

All the URLs in the UI have changed, and the public and admin pages work fine, only Icecast still shows all the old URLs pointing to the previously used domain.

**How to update (or have the system update) the URLs Icecast uses?**

**Comment by Vaalyn** [comment, marked as answer]
Have you tried rewriting the Icecast config via the `Restart Broadcasting` button yet?

**Comment by Moonbase59** [comment]
Thanks for the suggestion. Will "Restart Broadcasting" always rewrite the Icecast config? That would be good to know.

I had to shutdown my station today for a moment, due to work on the server, and when I restarted it, the Icecast URLs were correct. Unfortunately, I’m unsure if I did a "Restart Broadcasting" in the meantime.

**Reply by Vaalyn** [reply]
Yes, the `Restart Broadcasting` should always trigger a config write before the actual restart.
